ksqldb-engine/src/main/java/io/confluent/ksql/analyzer/PullQueryValidator.java

```diff
@@ -62,6 +66,14 @@
       Rule.of(
           analysis -> !analysis.getRefinementInfo().isPresent(),
           "Pull queries don't support EMIT clauses."
+      ),
+      Rule.of(
+          analysis -> !PullQueryValidator.disallowedColumnNameInSelectClause(analysis),
+          "Pull queries don't support ROWPARTITION or ROWOFFSET in select clauses"
```

Comment:
Because the string has no knowledge of the predicate's result, I can't really customize the message to include the exact column that is offending- or rather, I don't see a clean way of including it. I see a small extensibility and message clarity issue, probably worth fixing if there is a clean way, probably not if not. WDYT?